examples/all-clusters-minimal-app/asr/BUILD.gn

```diff
@@ -72,6 +72,8 @@ asr_executable("clusters_minimal_app") {
 
   sources = [
     "${chip_root}/examples/all-clusters-app/all-clusters-common/src/bridged-actions-stub.cpp",
+    "${chip_root}/examples/all-clusters-app/all-clusters-common/src/operational-state-delegate-impl.cpp",
```

Comment:
maybe you should have taken the goal to add this into the minimal. not sure about the policies regarding minimal vs regular

Reply:
No, the impls should be in examples.  That said, the minimal and non-minimal all-clusters-app can share code...